lib/models/ObjectMD.ts

```diff
@@ -96,6 +96,7 @@ export type ObjectMDData = {
     // the master is set as a placeholder and gets updated with the new latest
     // version data after a certain amount of time.
     isPHD?: boolean;
+    bucketOwnerId?: string;
```

Comment:
document here the bucketOwnerId, and document the `owner-id` as well.

Should we also have a `bucket-owner-display-name` ?

Reply:
I think we should be fine without `bucket-owner-display-name`.